src/components/organisms/CreditsHistorySection/CreditsHistorySection.spec.tsx

```diff
@@ -0,0 +1,48 @@
+import { render } from '@testing-library/react'
+
+import CreditsHistorySection from '@/components/organisms/CreditsHistorySection/CreditsHistorySection'
+import { TestOvermindWrapper } from '@/test/TestOvermindWrapper'
+import { MockData } from '@/test/TestOvermindMockData'
+
+Object.defineProperty(window, `matchMedia`, {
+  writable: true,
+  value: jest.fn().mockImplementation((query) => ({
+    matches: false,
+    media: query,
+    onchange: null,
+    addListener: jest.fn(), // deprecated
+    removeListener: jest.fn(), // deprecated
```

Comment:
Deprecated? So why is it there? Is it from a copy/paste?

Reply:
this chunk of code is removed as it is no longer required. it was used previously with the useBreakpointValue hook